### ISSUE TITLE
Show on Map opens map view in left sidebar, removes the "Files" icon and requires Obsidian restart to fix it.

### ISSUE THREAD

**Issue by rolandhesz** [esm7/obsidian-map-view#137]
When I right-click on a note and select "Show on Map" the map view opens in the left sidebar, removew the Files icon from the top, replacing it with it's own note icon.

![image](https://user-images.githubusercontent.com/13273796/217606170-6d0116f7-b033-41a6-8eeb-1c2f64653d56.png)

![image](https://user-images.githubusercontent.com/13273796/217606323-99df839f-6be3-48e3-9c31-a6cbaf45ef0f.png)

Now I can't open the file tree.

After restarting Obsidian the "Files" icon comes back.
![image](https://user-images.githubusercontent.com/13273796/217606587-c85cab25-94d1-4a43-9e7a-4ed4d44ec5b5.png)

I can't find any option in the settings that would change the behaviour to "Open a new tab in the main window and show the Map View there", like this:
![image](https://user-images.githubusercontent.com/13273796/217607300-c5abdbe5-dbe1-496f-857b-771ce3395ced.png)

I tried to see it if this happens because of the Minimal theme, but the same happens with the Default theme too.
Any chance of fixing this, or showing a desctiption onhow to change this behaviour?

**Comment by esm7** [comment]
I'll fix it as soon as I manage to work on the plugin. In the meantime, under the plugin settings try a different setting under Pane & Tab Usage -> Default mode for opening Map View.

**Comment by rolandhesz** [comment]
> 

Thank you, picking "Open in new pane" solved the problem. :)

**Comment by rolandhesz** [comment]
@esm7 I'm using 3.0.1, but I can't reproduce it anymore - I followed your advice and changed the behaviour, and I have no idea what the settings were when I ran into the issue.
I tried a few but still can't reproduce. Given that I haven't updated the plugin since February, I think it is likely that there was some clash with some other plugin, or with Obsidian, and that clash went away since then.

I should have saved the whole setup when I found the bug, but I didn't.
